contracts/netmap/contract.go

```diff
@@ -178,6 +185,13 @@ func _deploy(data any, isUpdate bool) {
 			setConfig(ctx, "UseNodeV2", []byte{0})
 		}
 
+		if version < 22_000 {
+			curEpoch := storage.Get(ctx, snapshotEpoch).(int)
+			curEpochHeight := storage.Get(ctx, snapshotBlockKey).(int)
+			storage.Put(ctx, append([]byte{epochIndexKey}, fourBytesBE(curEpoch)...), std.Serialize(epochItem{height: curEpochHeight}))
```

Comment:
Time info isn't present (can be retrieved from block header), but I doubt it's critical, the next epoch will have it anyway.